src/manifest.json

```diff
@@ -1,6 +1,7 @@
 {
-    "name": "GLIB-ChromeExt",
-    "version": "1.5",
+  "name": "GLIB-ChromeExt",
+  //"key": "glib", Used for local dev
+  "version": "1.5",
```

Comment:
So I added this commented out value b/c i needed it in local dev to have the correct callback url for auth. Spacing ugh